sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImpl.java

```diff
@@ -100,229 +102,197 @@ public TextAnalyticsClientImpl(HttpPipeline httpPipeline) {
-     *
-     * @param input Collection of documents to analyze.
-     * @param context The context to associate with this operation.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @return a Mono which performs the network request upon subscription.
-     */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<SimpleResponse<EntitiesResult>> entitiesRecognitionPiiWithRestResponseAsync(MultiLanguageBatchInput input, Context context) {
-        final String modelVersion = null;
-        final Boolean showStats = null;
-        return service.entitiesRecognitionPii(this.getEndpoint(), modelVersion, showStats, input, context);
-    }
-
-    /**
-     * Entities containing personal information
-     * The API returns a list of entities with personal information (\"SSN\", \"Bank Account\" etc) in the document. For the list of supported entity types, check &lt;a href="https://aka.ms/tanerpii"&gt;Supported Entity Types in Text Analytics API&lt;/a&gt;. See the &lt;a href="https://aka.ms/talangs"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.
-     *
-     * @param input Collection of documents to analyze.
+     * The API returns a list of recognized entities with links to a well-known knowledge base. See the &lt;a href="https://aka.ms/talangs"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.
+     * 
+     * @param input Contains a set of input documents to be analyzed by the service.
      * @param modelVersion (Optional) This value indicates which model will be used for scoring. If a model-version is not specified, the API should default to the latest, non-preview version.
      * @param showStats (Optional) if set to true, response will contain input and document level statistics.
-     * @param context The context to associate with this operation.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @return a Mono which performs the network request upon subscription.
+     * @throws TextAnalyticsErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<SimpleResponse<EntitiesResult>> entitiesRecognitionPiiWithRestResponseAsync(MultiLanguageBatchInput input, String modelVersion, Boolean showStats, Context context) {
-        return service.entitiesRecognitionPii(this.getEndpoint(), modelVersion, showStats, input, context);
+    public Mono<SimpleResponse<EntityLinkingResult>> entitiesLinkingWithResponseAsync(MultiLanguageBatchInput input, String modelVersion, Boolean showStats) {
```

Comment:
Why is Java diverging in naming here, for python we have it as RecognizeLinkedEntitiesResult

Reply:
autogenerated type. We do internal mapping.